decs.d.ts

```diff
@@ -135,4 +135,11 @@ declare module '@gridsuite/commons-ui' {
     }
 
     export const FlatParameters: FunctionComponent<FlatParametersProps>;
+
+    export const elementType = {
```

Comment:
Why do you put this here ?

Reply:
This file is to define TypeScript interfaces when we import JavaScript libraries (commonly commons-ui)